querydsl-sql/src/test/java/com/querydsl/sql/types/TypeTest.java

```diff
@@ -45,7 +45,29 @@
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-        if (method.getName().startsWith("get")) {
+        if (method.getName().equals("wasNull")) {
```

Comment:
I wasn't aware of `Objects.equal`. I should have known guava would have a helper method for this!